src/gui/tray/usermodel.cpp

```diff
@@ -495,47 +496,81 @@ bool User::isUnsolvableConflict(const SyncFileItemPtr &item) const
 
 void User::processCompletedSyncItem(const Folder *folder, const SyncFileItemPtr &item)
 {
+    const auto fileActionFromInstruction = [](const int instruction) {
+        if (instruction == CSYNC_INSTRUCTION_REMOVE) {
+            return QStringLiteral("file_deleted");
+        } else if (instruction == CSYNC_INSTRUCTION_NEW) {
+            return QStringLiteral("file_created");
+        } else if (instruction == CSYNC_INSTRUCTION_RENAME) {
+            return QStringLiteral("file_renamed");
+        } else {
+            return QStringLiteral("file_changed");
+        }
+    };
+
+    const auto messageFromFileAction = [](const QString fileAction, const QString fileName) {
+        if (fileAction == "file_renamed") {
```

Comment:
Why remove `QStringLiteral()` around literals?

Reply:
I can add QStringLiterals around the conditionals but the QObject::tr only accepts char pointers so those have to stay as literals